File: AuthServer.API/Models/Author.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace AuthServer.API.Models
{
    public class Author
    {
        [Key]
        public Guid Id { get; set; }

        public string FirstName { get; set; }
        public string LastName { get; set; }
        public virtual ICollection<BookAuthor> Books { get; set; }
    }
}

File: AuthServer.SPA/ClientApp/app/models/author.ts
import {Book} from "./book";

export class Author {
    id: string;
    firstName: string;
    lastName: string;
    books: Book[];

    constructor(id: string, firstName: string, lastName:string){
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
    }
}